.github/workflows/jekyll-gh-pages.yml

```diff
@@ -0,0 +1,91 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+    paths: ["docs/**/*.markdown"]
```

Comment:
why limiting to ".markdown" only?

Reply:
As discussed: change to all in `docs` dir